### ISSUE TITLE
auto detect fat32 in fist run

### ISSUE THREAD

**Issue by GoogleCodeExporter** [WhiteNeo/wiiflow#143]
```
When i haven´t a wiiflow.ini, WiiFlow say WBFS -1, and i go in settings 
(And the mesage appear again 2 times more.
Can you autodetect fat32 on the first run, with that it would be easy to 
the new people...
```

Original issue reported on code.google.com by `rvmn95` on 20 Mar 2010 at 5:12

**Comment by GoogleCodeExporter** [comment]
```
that was solved in  wiiflow 2.0??
```

Original comment by `rvmn95` on 19 Jun 2010 at 11:43

**Comment by GoogleCodeExporter** [comment]
Original comment by `Miigotu` on 22 Jun 2010 at 8:16
- Changed state: **Fixed**
